draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -860,6 +862,10 @@ cookie        = cookie-string
 cookie-string = cookie-pair *( ";" SP cookie-pair )
 ~~~
 
+Servers MUST be tolerant to potentially receiving multiple cookie
```

Comment:
"to" or "of"?

Reply:
I'd write "of". And I'd probably drop "potentially receiving", since it seems clear from context.

That said, there's nothing wrong with this, just personal preference.

Reply:
SG